prime-dotnet-webapi/Models/SiteRegistration/IndividualDeviceProvider.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("IndividualDeviceProvider")]
+    public class IndividualDeviceProvider : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public int CommunitySiteId { get; set; }
```

Comment:
Dont need `[Required]` on the FK property, it should go on the `CommunitySite` property. I'm pretty sure you actually don't need this at all, since this is an `int` property rather than a `int?` so EF knows that `CommunitySite` is required.